### ISSUE TITLE
Boost 1.78 Fiber and Stacktrace libs not installed

### ISSUE THREAD

**Issue by gpmcq15** [boostorg/boost#594]
Hello,

I'm a relatively new user of Boost, so please let me know if I haven't provided sufficient info or if this issue belongs elsewhere. I've been attempting to build Boost 1.78 locally (Ubuntu 20.04, WSL) from the tar download:
```
[boost]$ sha256sum boost_1_78_0.tar.gz
94ced8b72956591c4775ae2207a9763d3600b30d9d7446562c552f0a14a63be7  boost_1_78_0.tar.gz
```

Using
```
[boost]$ gcc --version
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
```

I noticed that 4 libs appeared to be missing when I tried to build everything ('libboost_fiber`, `libboost_stacktrace_noop`, `libboost_stacktrace_addr2line`, and `libboost_stacktrace_basic`). As a smaller reproduction, when I execute the following (atomic as a "working" example plus the ones I found to be missing):
```
[boost]$  tar -xvf boost_1_78_0.tar.gz && cd boost_1_78_0 && ./bootstrap.sh --with-toolset=gcc --with-libraries=atomic,fiber,stacktrace && ./b2 -j8 cxxflags=-std=c++17
```
Then `./stage/lib` only appears to get the `atomic` libs:
```
[boost_1_78_0]$ ls ./stage/lib
cmake  libboost_atomic.a  libboost_atomic.so  libboost_atomic.so.1.78.0
```

As a sanity check, I also tried a comparable command on `boost_1_77_0.tar.gz` and found the `fiber` and `stacktrace` libs to be available in `./stage/lib` as expected post build.

Am I doing anything incorrectly here / is this a known issue?

**Comment by gpmcq15** [comment]
Edit: maybe related to https://github.com/boostorg/fiber/issues/286 but didn't see any sort of resolution there really

**Comment by olk** [comment]
I believe this issue is related to the cmake stuff.

**Comment by moretromain** [comment]
I also reproduce this issue with windows/msvc (14.29): `b2 --with-fiber` outputs static libs in `\bin.v2\libs\fiber\build\msvc-14.2\release\link-static\threading-multi\libboost_fiber-vc142-mt-x64-1_78.lib` and `bin.v2\libs\fiber\build\msvc-14.2\debug\link-static\threading-multi\libboost_fiber-vc142-mt-gd-x64-1_78.lib` but the libraries are not installed into `stage\lib` as expected.

I tried different variants and build options (link=static/shared and various debug/release/runtime-link combos), but boost.fiber is the only library that exhibits this behavior, no matter the build configuration (I didn't check StackTrace though).

The issue does not occur with 1.77.0 when using the exact same build commands.

I tried to dig into the cmake stuff to see if I could find something obvious there that could explain the differences between libs, but after quite some time I couldn't find anything. I'm not familiar enough with boost.build to effectively debug this issue, but I'd be happy to help if someone could point me into the right direction, as I already spent quite some time trying to update my codebase to boost 1.78.

**Comment by brad0** [comment]
This is probably related to the  known issues listed item..

https://www.boost.org/patches/1_78_0/0001-b2-fix-install.patch

**Comment by mclow** [comment]
Can you try the 1.79.0 beta and confirm that this is fixed there?  https://boostorg.jfrog.io/artifactory/main/beta/1.79.0.beta1/source/

**Comment by brad0** [comment]
> Can you try the 1.79.0 beta and confirm that this is fixed there? https://boostorg.jfrog.io/artifactory/main/beta/1.79.0.beta1/source/

The errata patch that went out for 1.78.0 has not been integrated into a release and pushed to the 1.79 release cycle until after beta1 went out. The fix should be in the next 1.79 release.
